generator/README.md

```diff
@@ -78,7 +78,7 @@ modules:
                                     # Can be noAuthNoPriv, authNoPriv or authPriv.
       password: pass  # Has no default. Also known as authKey, -A option to NetSNMP.
                       # Required if security_level is authNoPriv or authPriv.
-      auth_protocol: SHA  # MD5 or SHA, defaults to SHA. -a option to NetSNMP.
+      auth_protocol: SHA  # MD5 or SHA, defaults to MD5. -a option to NetSNMP.
```

Comment:
It should say MD5 on the left too.

Reply:
As this is a small edit, I just amended the last commit.